src/transformers/modeling_t5.py

```diff
@@ -1050,6 +1091,12 @@ def __init__(self, config):
 
         self.init_weights()
 
+    def tie_weights(self):
+        # for COVID-19 3B input & output
+        # embeddings don't seem to be shared
+        # word embeddings are not tied in Reformer
```

Comment:
why is reformer relevant?

Reply:
Typo! It was copy pasted from Reformer. Will not clean this PR though until we want to integrate it